mappings/net/minecraft/util/registry/DataPackLoadedRegistries.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_5458 net/minecraft/util/registry/DataPackLoadedRegistries
+	FIELD field_25925 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25926 REGISTRIES Lnet/minecraft/class_2378;
+	FIELD field_25927 CONGINURED_SURFACE_BUILDER Lnet/minecraft/class_2378;
+	FIELD field_25928 CONFIGURED_CARVER Lnet/minecraft/class_2378;
+	FIELD field_25929 CONFIGURED_FEATURE Lnet/minecraft/class_2378;
+	FIELD field_25930 CONFIGURED_STRUCTURE_FEATURE Lnet/minecraft/class_2378;
+	FIELD field_25931 STRUCTURE_PROCESSORS Lnet/minecraft/class_2378;
+	FIELD field_25932 STRUCTURE_POOL Lnet/minecraft/class_2378;
+	FIELD field_25933 BIOME Lnet/minecraft/class_2378;
+	FIELD field_25934 DEFAULT_VALUE_SUPPLIERS Ljava/util/Map;
+	FIELD field_25935 ROOT Lnet/minecraft/class_2385;
+	METHOD method_30559 init ()V
+	METHOD method_30560 set (Lnet/minecraft/class_2378;ILjava/lang/String;Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 0 registry
+		ARG 1 rawId
+		ARG 2 id
+		ARG 3 object
+	METHOD method_30561 add (Lnet/minecraft/class_2378;Ljava/lang/String;Ljava/lang/Object;)Ljava/lang/Object;
```

Comment:
Maybe `register`, for consistency with `Registry`?

Reply:
Hmm, this calls `MutableRegistry#add`, hence I named that. Wonder why that is mutable registry add in the first place